bin/bootstrax

```diff
@@ -328,25 +320,19 @@ def main():
 
 def main_loop():
     """Infinite loop looking for runs to process"""
-    global state_doc_id
 
     # Wait for mongo connection
     ping_dbs()
 
     # Ensure we're the only bootstrax on this host
     any_running = list(bs_coll.find({'host': hostname}))
```

Comment:
Depending on how large the collection cap size is, this could give a cursor with a lot of documents. A possible issue this could cause (other than wasted CPU cycles) is that we could spend time trying to kill a task that's already been killed. Possibly sort by descending `_id` and take the most recent 2 or 3?

Reply:
Good point. How about this:
I add a extra time field to the eb_montitor. If the current time is within say 60 s I'll update the latest _id in the collection. Otherwise I start a new one. This of course depends on what we want to show exactly on the website.

By the way the collection is capped to 36288000 B (36 MB) and 181440 files.

Reply:
Documents in a capped collection can't be updated, unless we implement the cap as a TTL index (which is how the system monitor and readout status collections are handled).

Also, does it make sense to update the query to only look for other pids? `{'pid' : {'$ne' : os.getpid()}}`?

Reply:
I have set 'eb_monitor' as a TTL collection (very short time of now to check it is working as desired) when confirmed I'll increase the time. 

Thx for the query suggestion!